### ISSUE TITLE
Show obstacles in the map

### ISSUE THREAD

**Issue by EinEinfach** [EinEinfach/CaSSAndRA#20]
History of points where the mower has failed or collided with an obstacle.
It would be interesting to know where the machine has failed while it is cutting or at the end of the task. In such a way that you can physically review the points where it has occurred and be able to solve them.
![image](https://github.com/EinEinfach/CaSSAndRA/assets/26028367/dd46162f-c8d3-4587-be61-defc85254dac)

**Comment by disaster123** [comment]
+1

**Comment by EinEinfach** [comment]
![photo1692969878](https://github.com/EinEinfach/CaSSAndRA/assets/26028367/12b8de1c-ef65-4184-8045-050bbd1029c2)

https://github.com/EinEinfach/CaSSAndRA/commit/a7e43fc6f18f09d0e57d90d5b96a52eb098506bc

**Comment by EinEinfach** [comment]
Is that adjustable in Sunray FW how long obstacle coordinates will be kept

**Comment by disaster123** [comment]
No. In my fw they stay forever. In the default sunray code they get reset on each target reach

**Comment by disaster123** [comment]
@EinEinfach if the mower Code removes an or multiple obstacles - it seems cassandra does not remove them. I still see the obstacles in the map even the code has cleared the obstacles in fw.

**Comment by EinEinfach** [comment]
Yes, that's correct. Cassandra keeps all obstacles. You can remove it by clicking cancel button. But I'll rebuild this feature, Cassandra slows down if amount of obstacles goes up. I don't what to to. Maybe I'll just synchronize the shown obstacles with sunray fw.

**Comment by disaster123** [comment]
yes ok - cassandra gets pretty slow with the obstacles - currently i've 60 obstacles.... on sunray it seems it fetches those in background and does it not very often... like every 30s or so

**Comment by EinEinfach** [comment]
Seems to work now
